client/controller/api/apps.go

```diff
@@ -6,7 +6,7 @@ type App struct {
 	ID      string `json:"id"`
 	Owner   string `json:"owner"`
 	Updated string `json:"updated"`
-	URL     string `json:"url"`
+	URL     string
```

Comment:
Done - Why does it have to be defined as `json` to be ignored right away oppose to a straight up`string` ?